rllib/env/multi_agent_env_runner.py

```diff
@@ -346,7 +348,7 @@ def _sample(
                         metrics_prefix_key=(MODULE_TO_ENV_CONNECTOR,),
                     )
                 # In case all environments had been terminated `to_module` will be
-                # empty and no actions are needed b/c we reset all environemnts.
+                # empty and no actions are needed b/c we reset all environments.
```

Comment:
I wonder why this happens now that the `to_module` is `None`. Can we debug another round and see where this happens? Then check the autoreset and the connector run (I know this is complex). 

What should happen is: env resets automatically; init obs goes through the `to_module` connector pipeline and produces `to_module` which can in turn passed through the module and the `to_env` pipeline to produce an action.